trtools/compareSTR/compareSTR.py

```diff
@@ -713,12 +736,64 @@ def bad_range():
             common.WARNING("Cannot have range portion of --region "
                            "start-end where end <= start")
             return 1
-    except ValueError: # an int() cast failed
+    except ValueError:  # an int() cast failed
         return bad_range()
 
     return 0
 
 
+def handle_overlaps(records: List[Optional[trh.TRRecord]], chrom_indices: List[int], min_chrom_index: int) -> bool:
+    """
+        This function determines whether (two) records in list are comparable
+        Currently only works with record lists which are two records long
+
+        Parameters
+        ----------
+        records: List[Optional[trh.TRRecord]]
+            List of TRRecords whose comparability is to be determined. If any of them is None,
+            they are not comparable
+        chrom_indices: List[int]
+            List of indices of chromosomes of current records
+        min_chrom_index: int
+            Smallest index in chrom_indices. All records should have the same chrom_index,
+            otherwise they are not comparable
+
+        Returns
+        -------
+        comparable: bool
+            Result, that says whether records are comparable
+        """
+    assert len(records) == 2
+    min_overlap = 1.0
+
+    if any(record is None for record in records):
+        return False
+
+    left, right = records[0], records[1]
+    if chrom_indices[0] != chrom_indices[1] or \
+            chrom_indices[0] != min_chrom_index or \
+            chrom_indices[1] != min_chrom_index:
+        return False
+
+    left_start, left_end = left.pos, left.end_pos
+    right_start, right_end = right.pos, right.end_pos
+
+    overlap = min(left_end, right_end) - max(left_start, right_start)
+    # This calculation contains max() - 1 to compensate
```

Comment:
I don't think this is the right implementation. Shouldn't we be adding one to overlap above instead of subtracting one from the denominator?

Reply:
Yes, you are right, this implementation only calculates the right score for completely overlapping sequences, otherwise, it would calculate a score that is slightly lower than intended. Thank you for pointing that out!